projects/control-service/projects/pipelines_control_service/src/integration-test/java/com/vmware/taurus/service/deploy/DataJobDeploymentCrudITV2.java

```diff
@@ -159,6 +159,12 @@ public void testSynchronizeDataJob() throws Exception {
     Assertions.assertNotEquals(lastDeployedDateInitial, lastDeployedDateShouldBeChanged);
     Assertions.assertNotEquals(
         deploymentVersionShaShouldNotBeChanged, deploymentVersionShaShouldBeChanged);
+
+    // Deletes deployment
```

Comment:
We should really considering refactoring those long test methods into multiple methods and using @Order annotation to dictate the order . 

It's not something you need to do now, but you can add a TODO if you want for next developer to consider doing this.

Reply:
Yes, this is planned.